src/modules/product/controller/productController.ts

```diff
@@ -46,4 +46,21 @@ const createShop = async (req: ExtendRequest, res: Response) => {
       });
     }
   };
-export default { createProduct, createShop }
\ No newline at end of file
+
+  const updateProductAvailability = async (req: ExtendRequest, res: Response) => {

```

Comment:
let's change this function **`updateProductAvailability`** to **`updateProductStatus`**